homeworks/alina.yaremenko_AlinaYaremenko/2-logic/logic.js

```diff
@@ -0,0 +1,42 @@
+const firstNumber = 505;
+const secondNumber = 404;
+
+const multiply = '*';
+const divide = '/';
+const add = '+';
+const subtract = '-';
+
+const operation = '*';
+
+if (operation === subtract) {
+    const result = firstNumber - secondNumber;
+    console.log(result);
+} else if (operation === multiply) {
+    console.log(firstNumber * secondNumber);
+} else if (operation === divide) {
+    console.log(firstNumber / secondNumber);
+} else if (operation === add) {
+    console.log(firstNumber + secondNumber);
+}
+
+switch (operation) {
```

Comment:
Please, create two separate files for each logic: one for `if`, the second one for `switch-case`